<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>CConnectDlgBase</class>
    <widget class="QDialog" name="CConnectDlgBase">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>521</width>
                <height>200</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Connection Setup</string>
        </property>
        <property name="windowIcon">
            <iconset resource="resources.qrc">
                <normaloff>:/png/main/res/fronticon.png</normaloff>:/png/main/res/fronticon.png
            </iconset>
        </property>
        <property name="styleSheet">
            <string/>
        </property>
        <property name="sizeGripEnabled">
            <bool>true</bool>
        </property>
        <layout class="QVBoxLayout">
            <item>
                <layout class="QHBoxLayout">
                    <item>
                        <widget class="QLabel" name="lblServerAddr">
                            <property name="text">
                                <string>Server</string>
                            </property>
                            <property name="alignment">
                                <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                            </property>
                        </widget>
                    </item>
                    <item>
                        <widget class="QLineEdit" name="edtServerAddress"/>
                    </item>
                </layout>
            </item>

<item>
    <layout class="QHBoxLayout">
        <item>
            <widget class="QLabel" name="lblName">
                <property name="text">
                    <string>Name</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
            </widget>
        </item>
        <item>
            <widget class="QLineEdit" name="edtName"/>
        </item>
        <item>
            <widget class="QPushButton" name="butConnect">
                <property name="text">
                    <string>&amp;Connect</string>
                </property>
                <property name="default">
                    <bool>true</bool>
                </property>
            </widget>
        </item>
    </layout>
</item>

<item>
    <layout class="QHBoxLayout">
        <item>
            <widget class="QTextEdit" name="txtPlayers">

            </widget>
        </item>
    </layout>
</item>

<item>
    <layout class="QHBoxLayout">
        <item>
            <widget class="QComboBox" name="cmbPlayers">

            </widget>
        </item>
        <item>
            <widget class="QSlider" name="sliderVolume">

            </widget>
        </item>
    </layout>
</item>

<item>
    <layout class="QHBoxLayout">
        <item>
            <widget class="QLabel" name="lblPlayer">
                <property name="text">
                    <string>PlayerId</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
            </widget>
        </item>
        <item>
            <widget class="QLineEdit" name="edtPlayerId"/>
        </item>
        <item>
            <widget class="QLabel" name="lblVol">
                <property name="text">
                    <string>Vol</string>
                </property>
                <property name="alignment">
                    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
            </widget>
        </item>
        <item>
            <widget class="QLineEdit" name="edtVolume"/>
        </item>
        <item>
            <widget class="QPushButton" name="butSetVolume">
                <property name="text">
                    <string>&amp;SetVolume</string>
                </property>
                <property name="default">
                    <bool>false</bool>
                </property>
            </widget>
        </item>
    </layout>
</item>

<item>
    <layout class="QHBoxLayout">
        <item>
            <spacer>
                <property name="orientation">
                    <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                    <size>
                        <width>10</width>
                        <height>10</height>
                    </size>
                </property>
            </spacer>
        </item>
        <item>
            <widget class="QPushButton" name="but64">
                <property name="text">
                    <string>Buff64</string>
                </property>
                <property name="default">
                    <bool>false</bool>
                </property>
            </widget>
        </item>
        <item>
            <widget class="QPushButton" name="but128">
                <property name="text">
                    <string>Buff128</string>
                </property>
                <property name="default">
                    <bool>false</bool>
                </property>
            </widget>
        </item>
        <item>
            <widget class="QPushButton" name="but256">
                <property name="text">
                    <string>Buff256</string>
                </property>
                <property name="default">
                    <bool>false</bool>
                </property>
            </widget>
        </item>

        <item>
            <widget class="QPushButton" name="butPracticeMode">
                <property name="text">
                    <string>&amp;PracticeMode</string>
                </property>
                <property name="default">
                    <bool>false</bool>
                </property>
            </widget>
        </item>
    </layout>
</item>
        </layout>
    </widget>
    <resources>
        <include location="resources.qrc"/>
    </resources>
    <connections/>
</ui>
